actionlist/app/components/action_item_body.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  Button,
+} from 'react-native';
+
+const styles = StyleSheet.create({
+  actionContainer: {
+    backgroundColor: '#990000',
+    padding: 5,
+    borderRadius: 8,
+    width: 100,
+  },
+  bodyView: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+  },
+  routeContainer: {
+    backgroundColor: '#ffffff',
+    padding: 5,
+    borderRadius: 8,
+    borderWidth: 1,
+    borderColor: '#990000',
+    marginRight: 10,
+    width: 100,
+  },
+  actionButton: {
+    fontSize: 15,
+    color: '#ffffff',
+  },
+  routeButton: {
+    fontSize: 15,
+    color: '#990000',
+  },
+});
+
+const routeLog = () => ({});
+
+const takeAction = () => ({});
+
+const ActionItemBody = ({ actionRequested,
```

Comment:
If you wanted to get fancy, you can further destructure your props:
```es6
const ActionItemBody = ({
  actionRequested: [{ actionRequestedLabel: label }],
  initiator,
  lastApprovedDate,
  processType: [{ processTypeLabel: label }],
  processInstanceStatus: [{ processInstanceStatusLabel: label }] }) => (
  <View>
    <Text>{processTypeLabel}</Text>
    { /* ... */ }
  </View>
);
```
But it's not necessary (also, that's untested, I may have some syntax wrong there).